tests/test_disposables.py

```
@@ -0,0 +1,376 @@
+    assert bool(disposables) is False
+
+
+def test_non_empty_disposables_is_truthy():
+    mock = MockDisposable()
+    disposables = Disposables(mock)
+    assert disposables
+    assert bool(disposables) is True
+
+
+@mark.asyncio
+async def test_enter_with_no_disposables():
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)